iquip/apps/explorer.py

```diff
@@ -89,12 +113,20 @@ def openExperiment(self):
         experimentFileItem = self.explorerFrame.fileTree.currentItem()
         if experimentFileItem.childCount():
             return
-        experimentPath = experimentFileItem.text(0)
+        experimentPath = self.getFullPath(experimentFileItem)  # pylint: disable=unused-variable
```

Comment:
Why is this line necessary?

Reply:
It is a preparation step for opening the experiment builder.
For submitting or getting the arguments, we need to know the full path, not only the file name.
So I've implemented up to here first.

Reply:
In that case, please leave a TODO comment describing what is this variable for and how and when it will be used.

Reply:
Umm.. But there is already a TODO comment in the docstring of the method, so I think an additional comment is not necessary.
How about remaining it as it is?

Reply:
Oh, I did not see that. Then it seems alright!